contracts/scripts/_discoveryprovider-test-init.js

```diff
@@ -50,10 +50,21 @@ const printUsageAndThrow = () => {
 }
 
 /** Copies the contents of build/contracts to the outputPath */
-const copyBuildDirectory = (outputPath) => {
+const copyBuildDirectory = async (outputPath) => {
   let dir = path.join(__dirname, '..')
   let localTarget = path.join(dir, 'build/contracts')
-  fs.mkdirSync(outputPath, {recursive: true})
+
+  /* Creates build/contrats folder if folder is not present */
+  async function createContractsDir (dir) {
```

Comment:
is this a function inside copyBuildDIrectory? just to clarify

Reply:
yes it is!